lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -8,13 +8,33 @@ const { allAuthedUsersId, bucketOwnerActions, logId, publicId,
 const publicReadBuckets = process.env.ALLOW_PUBLIC_READ_BUCKETS ?
     process.env.ALLOW_PUBLIC_READ_BUCKETS.split(',') : [];
 
-function checkBucketAcls(bucket, requestType, canonicalID) {
+function checkBucketAcls(bucket, requestType, canonicalID, mainApiCall) {
+    // Same logic applies on the Versioned APIs, so let's simplify it.
+    const requestTypeParsed = requestType.endsWith('Version') ?
+        requestType.slice(0, -7) : requestType;
     if (bucket.getOwner() === canonicalID) {
         return true;
     }
+    // Backward compatibility
```

Comment:
what do you mean by "Backward compatibility": are you talking about keeping the "older" behavior (but i don't see the removed/updated code which would do it) of this code?

Reply:
Per this comment in the original PoC
https://github.com/scality/cloudserver/pull/5211/files#diff-773344fcfc0f64145ff95eb6eccd556588c19449dc2c263a2532d4030bf00dffR425

These APIs which have extra permissions are allowed by default to maintain backward compatibility with current behaviour as I understand it